xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/objects/classes/StringClass.java

```diff
@@ -123,4 +124,17 @@ public void displayEdit(StringBuffer buffer, String name, String prefix, BaseCol
         buffer.append(input.toString());
     }
 
+    @Override
+    public void displayView(StringBuffer buffer, String name, String prefix, BaseCollection object,
+        XWikiContext context)
+    {
+        BaseProperty property = (BaseProperty) object.safeget(name);
+        String content;
+        if (property != null) {
+            content = property.toText();
```

Comment:
I think I would have put the `buffer.append` directly in the if instead of going through a `content` variable but I guess it's a matter of taste.

Reply:
Else branch removed https://github.com/xwiki/xwiki-platform/pull/1315/files#diff-8574f36522a5e8eab9ad2acf469c1ef0R133